admin-dev/themes/new-theme/scss/pages/_cms_page.scss

```diff
@@ -0,0 +1,7 @@
+#cms_page {
+  .seo_preview {
+    .input-container {
+      max-width: calc(66.66%);
```

Comment:
@Progi1984 Franck, will this work on mobile? It should fill 100% of space there, no?